avi.wad.Organizer.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "avi.wad.Organizer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "organizer",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--filesystem=xdg-desktop",
+        "--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-music",
+        "--filesystem=xdg-pictures",
+        "--filesystem=xdg-videos",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
```

Comment:
Not needed.

Reply:
so I just remove 
`        "--talk-name=ca.desrt.dconf",`
`        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"`
then?

Reply:
because earlier on, I removed all dconf references but my app crashed and burned, which is why I added all dconf related arguments

Reply:
anyhow, I did it!